### ISSUE TITLE
"Evaluation failed: c" trying send video with sendMediaAsDocument = false

### ISSUE THREAD

**Issue by andresmv94** [pedroslopez/whatsapp-web.js#1816]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

when I'm sending a video as a gif "sendVideoAsGif in true" the _sendMessage_ function returns the error **"Error: Evaluation failed: c"**, but when I send the same video as a document "sendMediaAsDocument: true"  it sends successfully, any idea why?

simplified code:

```
const client = await new Client({
      authStrategy: new LocalAuth({ clientId: companyId }),
      authTimeoutMs: 60000,
      puppeteer: {
        executablePath: "C:/Program Files/Google/Chrome/Application/chrome.exe",
        args: [
            '--no-sandbox',
            '--disable-setuid-sandbox'
        ]
      }
    });
	
	await client.on("ready", async () => {
		...
		number_details = await client.getNumberId(number);
		
        if (number_details?._serialized) {
			const media = await MessageMedia.fromFilePath(
                path.join(mediaPath)
            );
			await client
              .sendMessage(number_details?._serialized, media, {
                caption: campaign.description,
                sendMediaAsDocument: true, // with this works fine
				/* sendVideoAsGif: true // with this return error (to test it, uncomment this line and comment the "sendMediaAsDocument: true" line or put it in false) */
            }).then((res) => {
				console.log("ok");
			}).catch((error) => {
				console.log(error);
			});
		}
	}
```

### Expected behavior

send videos as gif or as normal video and not as file

### Steps to Reproduce the Bug or Issue

run the code in the description 

### Browser Type

Google Chrome

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

    OS: Windows 10 (locally ubuntu 20 in production)
    Phone OS: Android
    whatsapp-web.js version 1.18.3
    Node.js Version v16.15.1

**Comment by andresindlp** [comment]
same issue

**Comment by andresmv94** [comment]
> same issue

@andresindlp , the problem was for the mp4 encode file, if your video file  is passed through a video to mp4 converter like https://video-converter.com/es/, the file is generated by correct mp4 format and the problems is solved, I actually dont know how check if the mp4 video is in the correct format, but my solution was pass the video by the converter and then send it. check if this is your same case.

**Comment by andresindlp** [comment]
@andresmv94
After some further investigation, Chromium seems to be the problem.

Here are some video files analysed using `ffprobe`:
<img width="736" alt="Screenshot 2022-11-21 at 20 57 18" src="https://user-images.githubusercontent.com/35582438/203148455-51ccb058-eca4-476f-9970-fff6a9e038e4.png">
<img width="738" alt="Screenshot 2022-11-21 at 20 58 12" src="https://user-images.githubusercontent.com/35582438/203148471-ccb177b5-2663-4c60-8d59-a85e4b07485c.png">
<img width="780" alt="Screenshot 2022-11-21 at 21 03 56" src="https://user-images.githubusercontent.com/35582438/203148479-4f7fe8c7-9511-4375-bac6-9de9c7bbec05.png">

Two of them are H264 while one of them is H265. None where able to be sent as normal videos (`sendMediaAsDocument: false`) by default.

When defining the client function, Chromium is chosen unless otherwise specified.
```javascript
const client = new Client({
    restartOnAuthFail: true,
    puppeteer: {
        headless: true,
        executablePath: "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome" // This has to be set to the Google Chrome binary, not Chromium as it is by default.
    },
    authStrategy: new LocalAuth({ clientId: "client" })
});
```
Going over [Puppeteers documentation](https://pptr.dev/faq/#q-what-features-does-puppeteer-not-support) I've found that **Google Chrome is needed in order to send H264 and AAC** (which all videos I've tried have). Even then, it doesn't seem to work all of the time, but it has been working just fine recently. I'll make sure to update this issue if it breaks again.
